labs/src/main/scala/org/scalalabs/basic/lab02/CollectionExercise.scala

```diff
@@ -76,6 +105,6 @@ object CollectionExercise04 {
    * To keep it simple it's ok to use String.split to extract all words of a sentence.
    */
   def calcLengthLongestWord(lines: String*): Int = {
-    error("fix me")
+    lines.map((line: String) => line.split(' ').map((word:String) => word.length).max).max
```

Comment:
This line of code is probably long enough to break into pieces, using intermediate variables.